SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -191,7 +199,9 @@ var splunkRumInitializeCalledTime = Date()
         if options != nil {
             srInit.setAttribute(key: "config_settings", value: options!.toAttributeValue())
         }
-        initalizeNetworkInstrumentation()
+        if options?.isNetworkInstrumentation == true{
```

Comment:
`if options?.networkInstrumentation {` - no need to compare with true, that's what `if` does.